### ISSUE TITLE
After running 100 rounds, the losses of training, validation, and testing are all around 0.20. Should we increase the number of training rounds?

### ISSUE THREAD

**Issue by algwang** [anasiri/BU-Mamba#5]

**Comment by anasiri** [comment]
Which model and dataset are you using? In our experiments, the validation loss on the BUSI dataset—depending on the initialization and split seed (as we ran our experiments multiple times)—ranged from around 0.2 to 0.5 by the end of training. You should also consider other metrics, such as AUC.

**Comment by algwang** [comment]
OK, thank you. I would like to ask, does the main.py file integrate training, validation and testing?

**Comment by anasiri** [comment]
Yes, the main.py does everything. Happy to help.
